packages/react/package.json

```diff
@@ -91,8 +91,7 @@
     "lint:fix": "eslint . --fix",
     "storybook": "storybook dev -p 6006",
     "storybook:static": "http-server ./storybook-static",
-    "copy:styles": "node copystyles.js",
-    "test": "NODE_OPTIONS=--experimental-vm-modules SKIP_PREFLIGHT_CHECK=true react-scripts test --watchAll=false --setupFilesAfterEnv ./src/setup.ts"
```

Comment:
NVM found commit that makes the change (chore(react): remove test script before tests migration). It would have been better if this change was done in the test migration PR.

Reply:
It was done to avoid unexpected pipeline failures